src/codegen/java.rs

```diff
@@ -372,10 +376,21 @@ fn render_any_of_refs(
 
     // Class declaration.
     let mut class_decl = format!("sealed class {name}()");
-    if let Some(inherits) = ctx.inherits.get(name) {
+    let has_parent = ctx.inherits.get(name).is_some();
+    if has_parent {
+        let inherits = ctx.inherits.get(name).unwrap();
         class_decl.push_str(&format!(" : {inherits}()"));
     }
```

Comment:
Generally best to avoid `.unwrap()` whenever possible in Rust as it might panic. It's a big win whenever you can write your code in a way without `.unwrap()` so that the compiler can prove that you're not referring to a `None` option somewhere you shouldn't be. Perhaps something like this?

```suggestion
    let parent = ctx.inherits.get(name);
    if let Some(parent) = parent {
        class_decl.push_str(&format!(" : {parent}()"));
    }
```

(But also see below. Do we need to do this check `has_parent` at all?)